rubygems-org-api.md

```diff
@@ -47,6 +48,19 @@ examples in the README. You can install the library with the command:
 
     gem install gems
 
+Rate Limits
+-----------
+
+To protect the RubyGems.org service from abuse, both intentionally and unintentionally, we have rate limits in place for some of our endpoints. Some endpoints may be cached by our CDN at times and therefore _may_ allow higher request rates. The following is a general guideline for the rate limit rules.
+
+* API and website: 10 requests per second
+* Dependency API: 15 requests per second
+* Website sign up, sign in, and forgot password: 100 requests per 10 minutes
```

Comment:
`/api/v1/api_key` is also limited to 100 requests per 10 minutes (https://github.com/rubygems/rubygems.org/commit/1d07853277d325545c626776db0ce65cc340c03d).